cardinal/server/handler/tx.go

```diff
@@ -31,24 +31,21 @@ type Transaction = sign.Transaction
 // PostTransaction godoc
 //
 //	@Summary		Submit a transaction to Cardinal
-//	@Description	Submit a transaction to Cardinal
+//	@Description	Submit a transaction to Cardinal / Create a Persona transaction to Cardinal
 //	@Accept			application/json
 //	@Produce		application/json
-//	@Param			txType	path		string	true	"label of the transaction that wants to be submitted"
-//	@Success		200		{object}	PostTransactionResponse
-//	@Failure		400		{string}	string	"Invalid transaction request"
+//	@Success		200	{object}	PostTransactionResponse
+//	@Failure		400	{string}	string	"Invalid transaction request"
+//
+// PostTransaction with label godoc
+//
 //	@Router			/tx/game/{txType} [post]
+//	@Param			txType	path	string	true	"label of the transaction that wants to be submitted"
 //
 // PostTransaction with Persona godoc
 //
-//	@Summary		Create a Persona transaction to Cardinal
-//	@Description	Create a Persona transaction to Cardinal
-//	@Accept			application/json
-//	@Produce		application/json
-//	@Param			txBody	body		Transaction	true	"Transaction details"
-//	@Success		200		{object}	PostTransactionResponse
-//	@Failure		400		{string}	string	"Invalid transaction request"
 //	@Router			/tx/persona/create-persona [post]
+//	@Param			txBody	body	Transaction	true	"Transaction details"
 func PostTransaction(
```

Comment:
FYI: I made these changes here to fix issue this swaggo issue:

Swaggo supports multiple `@Router` on a single function like the `PostTransaction` function here 
but multiple defined annotations like `@Summary`, and `@Accept`, `@Produce` with same value causing a double, overridden, and invalid swagger specification format.

- The swagger-diff checker complains about these duplicated array values:
```
swagger.json is not a valid Swagger specification:
- The property '#/paths//tx/game/{txType}/post/produces' contained duplicated array values
- The property '#/paths//tx/game/{txType}/post/consumes' contained duplicated array values
```
![image](https://github.com/Argus-Labs/world-engine/assets/29672212/886ad513-5b75-46b2-a233-19ae5515cbda)

- `@Sumarry` for `/tx/game/{txType}` router getting overridden by `/tx/persona/create-persona` 
example when importing it in Postman:
![image](https://github.com/Argus-Labs/world-engine/assets/29672212/94324516-516e-4b4a-80a7-7b0fd2a8df8f)


The proposed fix:
- remove duplicate `@Accept` `@Produce` annotation from `PostTransaction` function
- use generic `@Summary` values that can describe `/tx/game/{txType}`  and `/tx/persona/create-persona`, so it less confusing than being `/tx/game` described as create-persona.

**There's an alternative workaround by creating an additional function wrapper, but I don't think that necessary for this swagger use case --> https://github.com/swaggo/swag/issues/1095#issuecomment-1489594724